### ISSUE TITLE
Bash aliases not highlighting anymore

### ISSUE THREAD

**Issue by 3ximus** [akinomyoga/ble.sh#103]
Hi there,
It seems that this issue was introduced in 0860be0 

Before @ 9125795
![print_001](https://user-images.githubusercontent.com/9083012/117011433-be973180-ace5-11eb-8c8f-bd71e43c8351.png)

After @ 0860be0 
![print](https://user-images.githubusercontent.com/9083012/117011488-cc4cb700-ace5-11eb-92ff-2589271f0f71.png)

It seems that bash aliases highlighting are the only thing that stopped working correctly

**Comment by akinomyoga** [comment]
Thank you! It seems it doesn't reproduce in my environment. Can you provide me with the related settings of aliases in each session of 9125795 and 0860be0? `ble.sh` may interfere with existing alias settings so I'd like to compare the difference between the two sessions.

```bash
$ alias
```

**Comment by 3ximus** [comment]
These are my current alias settings:
```
alias ..='cd ..'
alias ...='cd ../../'
alias ....='cd ../../../'
alias alert='echo -ne '\''\a'\'''
alias aur-packages='pacman -Qm'
alias clip-in='xclip -in -selection clipboard'
alias clip-out='xclip -out -selection clipboard'
alias dd='dd status=progress'
alias diff='diff --color=auto'
alias dirs='dirs -v'
alias egrep='egrep --color=auto'
alias ga='git add .'
alias gc='git commit'
alias gd='git diff'
alias gdb='gdb -q'
alias git-lfs='git rev-list --objects --all | grep "$(git verify-pack -v .git/objects/pack/*.idx | sort -k 3 -n | tail -10 | awk '\''{print$1}'\'')"'
alias gl='git lol | less'
alias gla='git lol --all | less'
alias gp='git push'
alias grep='grep --color=auto'
alias grep-bin='grep -oUaP'
alias gs='git status'
alias htop-cpu='htop --sort-key=PERCENT_CPU'
alias htop-mem='htop --sort-key=PERCENT_MEM'
alias l.='ls -Fd .*'
alias la='ls -AF'
alias less='less -Mrix4'
alias list-functions='select i in `grep -P "[a-zA-Z0-9_\-]+(?=\(\))" -o $HOME/.bash/functions.sh`; do declare -f $i; break; done'
alias list-packages='expac -H M "%m\t%-30n" | sort -rh | less'
alias ll='ls -lF'
alias lla='ls -lFA'
alias load-keychain='[[ -f "${HOME}/.ssh/id_rsa" ]] && keychain "${HOME}/.ssh/id_rsa" --quiet && source "${HOME}/.keychain/${HOSTNAME}-sh"'
alias ls='ls --color=auto'
alias man-pipes='echo "Generic form #>/dev/null (# is 1 by default) or #>&# to send one fd to another"; echo "  2>&-        ----> #>&-  (close fd)"; echo "  |&          ----> 2>&1 |"; echo "  &>/dev/null ----> 1>/dev/null 2>&1"'
alias mount-ntfs='sudo mount -o uid=1000,gid=1000,dmask=027,fmask=137'
alias paccache-clean='paccache -rk1 && paccache -ruk0'
alias pacman='pacman --color=auto'
alias perl-regex='perl -p -i -e'
alias pip-upgrade='pip3 list --outdated --format=freeze | grep -v "^\-e" | cut -d = -f 1 | xargs -n1 sudo pip3 install -U'
alias plex='sudo systemctl start plexmediaserver.service'
alias plex-restart='sudo systemctl restart plexmediaserver.service'
alias plex-status='systemctl status plexmediaserver.service'
alias plex-stop='sudo systemctl stop plexmediaserver.service'
alias restart-kwin='kwin_x11 --replace &'
alias restart-plasmashell='killall plasmashell && kstart5 $_ &>/dev/null'
alias restart-tiling='qdbus org.kde.KWin /KWin reconfigure'
alias tmux-attach-session='tmux list-sessions; read -r -p "Attach to > " REPLY; tmux attach-session -t $REPLY'
alias tmux-kill-session='tmux list-sessions; read -r -p "Kill session > " REPLY; tmux kill-session -t $REPLY'
alias un='extract'
alias wget-directory='wget -r -np -nc -nd -k'
```

**Comment by akinomyoga** [comment]
Thank you for the information! Hmm, I can't reproduce with the same alias settings.

Ah, OK. I could reproduce it with `git@github.com:3ximus/dotfiles.git` master + ble.sh 0860be0. I'll fix it at my side.

**Comment by 3ximus** [comment]
you think its some problem with my blerc ?

**Comment by akinomyoga** [comment]
I pushed a fix 45b30a7.

> you think its some problem with my blerc ?

No, it was just a bug of `ble.sh`. I'm sorry for my confusing writing! I tried your settings just to reproduce the problem. I meant by "fix it at my side" that I would pin down the problem in `ble.sh` without additional help from you :)

**Comment by 3ximus** [comment]
perfect its fixed :+1: 
thank you !
